nomos-blend/network/src/core/behaviour.rs

```diff
@@ -73,6 +76,8 @@ pub enum Event {
     UnhealthyPeer(PeerId),
     /// A peer has been detected as healthy.
     HealthyPeer(PeerId),
+    /// Failed to receive a message from an edge node.
+    EdgeNodeFailedReception(FailureReason),
```

Comment:
I don't think we really need this, do we? Like, there's not really anything we can do about it anyway. It's more useful for the sender to know about this.